utils/ansible/add-runner.yml

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2023, Intel Corporation
+
+# This playbook is designed to add runner to GHA on pmem/pmdk.
+# Below examples how to use this file:
+# 1) locally
+# sudo ansible-playbook ./add-runner.yml --extra-vars
+# "testUser=pmdkuser runner_name=name labels=label1,label2 vars_gha=env1,env2 token=gha_token"
+# 2) remote
+# ansible-playbook -i <target_platform_ip_address>, add-runner.yml --extra-vars
+# "host=all ansible_user=root ansible_password=<password_for_root_on_target> testUser=pmdkuser runner_name=name labels=label1,label2 vars_gha=env1,env2 token=gha_token"
+#
+# Runner package version may be changed by: --extra-vars package_url=<url_to_package.tar.gz>
+
+#- hosts: "{{ host }}"
+- hosts: localhost
+  connection: local
+  vars:
+    testUser: pmdkuser
+    package_url: https://github.com/actions/runner/releases/download/v2.304.0/actions-runner-linux-x64-2.304.0.tar.gz
+    runner_folder: /home/{{ testUser }}/actions-runner
+    repo_url: https://github.com/pmem/pmdk
+    vars_list: "{{ vars_gha.split(',') }}"
+
+  tasks:
+    - name: "Create a runner folder"
+      file:
+        path: '{{ runner_folder }}'
+        state: directory
+        force: yes
+
+    - name: "Download and extract the installer"
+      unarchive:
+        src: '{{ package_url }}'
+        dest: '{{ runner_folder }}'
+        remote_src: yes
+
+    - name: "Change owner to {{ testUser }}"
+      shell: |
+        chown -R {{ testUser }} {{ runner_folder }}
+
+    - name: "Add runner to GHA"
+      shell: |
+        cd {{ runner_folder }}
+        ./config.sh --url {{ repo_url }} --token {{ token }} --name {{ runner_name }} --labels {{ labels }} --runnergroup Default --work _work
+      become: yes
+      become_user: '{{ testUser }}'
+
+    - name: "Install runner service"
+      shell: |
+        cd {{ runner_folder }}
+        ./svc.sh install
```

Comment:
> The service shall be installed for `pmdkuser`
It's necessary to install this service with root privileges. Otherwise, the service installation won't run.
But it's possible to run as pmdkuser. Is the below solution sufficient?

![image](https://github.com/grom72/pmdk/assets/103180784/537cf0d6-1613-4127-a97c-c502d2e2e29e)

Reply:
Yes, this is what I mean.

Reply:
Done.